### PR TITLE
Us 77 move robber when dice is 7

### PR DESCRIPTION
**Pull request by andrii-r**

### CODE REVIEW

File: src/main/java/catan/services/impl/PlayServiceImpl.java [hunk 1]
```diff
@@ -200,7 +203,8 @@ private void throwDice(GameBean game) {
             List<HexBean> hexesWithCurrentDiceValue = game.fetchHexesWithCurrentDiceValue();
             log.debug("Hexes with current dice value:" + hexesWithCurrentDiceValue);
             mainStageUtil.produceResourcesFromActiveDiceHexes(hexesWithCurrentDiceValue);
-        } else{
+        } else {
```
Comment (andrii-r) [inline src/main/java/catan/services/impl/PlayServiceImpl.java]: I would invert this logic

File: src/main/webapp/features/screens/game/dashboard/gameMap/ct-game-map.js [hunk 2]
```diff
@@ -42,6 +42,11 @@ angular.module('catan')
                         SelectService.select('edge', edgeId);
                     });
 
+                    element.on('click', DrawMapService.HEX_SELECTOR, function(event) {
```
Comment (andrii-r) [inline src/main/webapp/features/screens/game/dashboard/gameMap/ct-game-map.js:45]: what happens when I click on the robbed hex?

Comment (skisor) [inline src/main/webapp/features/screens/game/dashboard/gameMap/ct-game-map.js:45]: App will try to move robber to the same place and get error. In future, back-end should provide info about places which are available for moving robber here -- like in building of road/settlement/city.

Comment (skisor) [inline src/main/webapp/features/screens/game/dashboard/gameMap/ct-game-map.js:45]: I've created story for this

**Pull request merged**